src/OpenTelemetry.Instrumentation.AspNetCore/README.md

```diff
@@ -61,9 +61,9 @@ This instrumentation can be configured to change the default behavior by using
 
 This instrumentation by default collects all the incoming http requests. It
 allows filtering of requests by using `Filter` function in
-`AspNetCoreInstrumentationOptions`. This can be used to filter out any requests
-based on some condition. The Filter receives the `HttpContext` of the incoming
-request, and filters out the request if the Filter returns false or throws
+`AspNetCoreInstrumentationOptions`. This defines the condition for allowable
+requests. The Filter receives the `HttpContext` of the incoming
+request, and does not instrument the request if the Filter returns false or throws
```

Comment:
I'd try to avoid the word "instrument" here. Can we use "does not collect telemetry about the request" ?

Even if we are filtering this request, we still want to extract/propagate the context. If we say "does not instrument", it may lead to confusion of whether the context is propagated or not.
https://github.com/open-telemetry/opentelemetry-dotnet/issues/1564

Reply:
👍 Thanks for the clarification.
I will update.